apps/civicsignalblog/src/payload/access/canAccessApplication.js

```diff
@@ -1,7 +1,9 @@
 export default function canAccessApplication(user, searchString) {
+  // We are using this condition to only control what a user can see at a given time
   if (user) {
     const app = user.currentApp || user.defaultApp;
     return app === searchString.toLowerCase();
   }
-  return false;
+  // For APIs coming from external apps we are not going to use any restriction in reading
+  return true;
```

Comment:
Can't we use a api key or something?

Reply:
We could use an API key, it might not add much value (since we'll be reading from purely client side) but it's going to be for `readOnly` users